internal/metrics/daemon.go

```diff
@@ -191,10 +190,10 @@ func (md *metricsDaemon) startTarget(target *TargetMetric) {
 }
 
 // AddTarget adds a metrics scraping target with filtering
-func (md *metricsDaemon) AddTarget(targetName string, urls []string, interval time.Duration, allowList SampleFilter) {
-	target := NewTargetMetric(targetName, interval, urls, md.store, allowList)
-	log.Debugf("added target '%v' with the following urls: '%+v", targetName, urls)
+func (md *metricsDaemon) AddTarget(targetName string, scrapers Scrapers, interval time.Duration, allowList SampleFilter) {
 
+	log.Debugf("added target '%v' with the following urls: '%s'", targetName, scrapers)
```

Comment:
I think it should be scrapers instead of urls

Reply:
Updated. Please take a look.